db/migrate/20221020215132_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :email
```

Comment:
FYI: Sometimes we can add here different restrictions on the DB level, for example:
```suggestion
      t.string :email, null: false
```